linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_utmp/oval/shared.xml

```diff
@@ -0,0 +1,40 @@
+<def-group>
```

Comment:
I think you do not need to use this file. You can use the one from the template. The template is here:
https://github.com/ComplianceAsCode/content/tree/master/shared/templates/audit_rules_login_events
You can keep your custom remediations, because the template uses hardcoded key "login", where you are using the key "session". But the OVAL check from the template does not check for the key at all, cause it can be arbitrary. So you can use for example this in the rule.yml:
```
template:
    name: audit_rules_login_events
    vars:
        path: /var/log/utmp
    backends:
        ansible: "off"
        bash: "off"
```
This will use your custom remediations but it will use OVAL check from the template.
It helps to avoid code duplication.

Reply:
removed oval and added template in the rule.yml